resources/views/home.antlers.html

```diff
@@ -1,3 +1,24 @@
+
+<div class="bg-concrete relative py-50 xl:py-100">
+
+{{ first_image_content }}
+  <div class="container flex flex-wrap {{ if type == 'image'  && image_position == 'right' }} flex-row-reverse {{/if}}">
```

Comment:
there probably no need for `flex` on mobile
```suggestion
  <div class="container xl:flex {{ if type == 'image'  && image_position == 'right' }} flex-row-reverse {{/if}}">
```